os/arch/arm/src/s5j/mpu-reg.h

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#ifndef __ARCH_ARM_INCLUDE_CHIP_MPU_H
+#define __ARCH_ARM_INCLUDE_CHIP_MPU_H
+
```

Comment:
since it is chip specific, lets keep define also chip name e.g.__ARCH_ARM_INCLUDE_S5J_MPU_H